cookbooks/aws-parallelcluster-common/resources/cloudwatch/partial/_cloudwatch_common.rb

```diff
@@ -135,7 +135,7 @@ def package_path
       'CW_LOGS_CONFIGS_PATH' => config_data_path
     )
     command "#{cookbook_virtualenv_path}/bin/python #{validator_script_path}"
-  end
+  end unless redhat_ubi?
```

Comment:
Question: Why has it been necessary to add `unless redhat_ubi?`, `unless on_docker?` or change `redhat_ubi?` with `unless on_docker?`? I do not see in this PR any removal from a top level recipe of `unless virtualised?` that might have required to add the `unless on_docker?` to the specific command in that recipe.
It seems a reduction of the scope of the system test.

Is it due to this line?
```
raise "Init package #{node['init_package']} not supported." unless systemd? || on_docker?
```
(same question for other below)

Reply:
I am not sure how those parts worked before, and for some of them I am pretty positive they couldn't work. This is why I would like to move GH actions on EC2 instead of investing in a part which is not reliable and requires significant maintenance effort.